src/PATHSolver.jl

```diff
@@ -103,6 +102,11 @@ function remove_option_file()
   end
 end
 
+# Needed for compatibility with Julia v0.5 through v0.7
+@static if VERSION < v"0.7.0-"
+  pairs(x) = [(p[1] => p[2]) for p in x]
```

Comment:
By the way, I only added the call to `pairs(kwargs)` inside `options` to get around this issue on Julia nightly: https://github.com/JuliaLang/julia/issues/25177 but it looks like that issue will be resolved, so it's probably safe to just get rid of the call to `pairs()` inside `options` and get rid of this code as well.

Reply:
Thanks. Removed `pairs()`.